src/components/Editor/CustomExtensions/Variable/constants.js

```diff
@@ -0,0 +1 @@
+export const MAX_VARIABLE_NAME_DISPLAY_LENGTH = 15;
```

Comment:
Let's increase it to 25. There are a lot of variables that get truncated at 15.

```suggestion
export const MAX_VARIABLE_NAME_DISPLAY_LENGTH = 25;
```